sources/packages/backend/apps/queue-consumers/src/processors/schedulers/cas-integration/_tests_/cas-supplier-integration.scheduler.e2e-spec.ts

```diff
@@ -366,6 +368,115 @@ describe(describeProcessorRootTest(QueueNames.CASSupplierIntegration), () => {
     },
   );
 
+  it("Should create a new site and update the student CAS supplier when an active CAS supplier exists with no match addresses.", async () => {
```

Comment:
Even though the technical AC did not mention about unit test for the create SITE API call, 
I would recommend a unit test for create Site CAS API call similar to [cas.service.createSupplierAndSite.spec.ts](https://github.com/bcgov/SIMS/blob/4a2ac40046572ada2ac28f23ca86be087e397b97/sources/packages/backend/libs/integrations/src/cas/_tests_/unit/cas.service.createSupplierAndSite.spec.ts#L1)

But let me check what team thinks

@andrewsignori-aot @guru-aot @andrepestana-aot @sh16011993  thoughts?

Reply:
Please add the unit test to the new method.

Reply:
I agree too